tools/reviewer/web/src/app/dashboard/reviews-panel/diff/diff.component.ts

```diff
@@ -78,36 +78,48 @@ export class DiffComponent implements OnInit, OnDestroy {
   getBranchInfo(): void {
     this.localserverService
       .getBranchInfo(this.diff.getId(), this.diff.getWorkspace())
-      .subscribe(diffFilesResponse => {
-        this.branchInfo = diffFilesResponse;
+      .subscribe(branchInfo => {
+        this.branchInfo = branchInfo;
         this.file.setRepoId(this.branchInfo.getRepoId());
-        this.file.setCommitId(this.getCommitId(this.file, this.branchInfo));
+        this.setCommitId(this.file, this.branchInfo);
         this.getFileChanges(this.file);
       });
   }
 
   // Get id of commit, where the file is present
-  getCommitId(currentFile: File, branchInfo: BranchInfo): string {
+  setCommitId(currentFile: File, branchInfo: BranchInfo): void {
     for (const commit of this.branchInfo.getCommitList()) {
       for (const file of commit.getFileList()) {
         if (currentFile.getFilename() === file.getFilename()) {
-          return commit.getId();
+          currentFile.setCommitId(commit.getId());
```

Comment:
Should't the `commitId` already be in the file? Maybe we're forgetting to set it in the server?